src/guess.js

```diff
@@ -0,0 +1,50 @@
+import compareNumbers from './compare-numbers.js';
+import changeTC from './changeTC.js';
+
+
+//Reference DOM elements
+const button = document.getElementById('submit-guess');
+const numberGuess = document.getElementById('number-guess');
+const trialMessage = document.getElementById('high-low-right');
+const triesDisplay = document.getElementById('numberTries');
+const gameResult = document.getElementById('win-lose-result');
+
+const numberAnswer = Math.floor((Math.random() * 20) + 1);
+//console.log('Actual Answer: ' + numberAnswer);
```

Comment:
before you submit a lab, delete out any console.logs and comments - if you feel like you'll need them later, I recommend committing right before you delete them so that you can refer or revert back to the commit later.